apply_DDT.py

```diff
@@ -10,7 +10,7 @@
 import os, os.path as osp, glob, pickle, logging, warnings, json, math, re
 from time import strftime
 from collections import OrderedDict
-from common import create_DDT_map_dict, calculate_varDDT
+from common import create_DDT_map_dict, calculate_varDDT, apply_cutbased
```

Comment:
seems like `apply_cutbased` is not actually used in this file

Reply:
Thanks, yes, it's only for making the histograms or 'applying' the cuts + ddt

Reply:
...so since it's not used, removed the unneeded import?

Reply:
It came back, must have been during the rebase. Hopefully it stays gone as I address the other comments.